### DISCUSSION TITLE
1.5.10

### DISCUSSION THREAD

**Discussion by stnguyen90** [appwrite/appwrite#8524] [Show and tell]
### Notable changes

* Bump console to version 4.3.30 by @stnguyen90 in https://github.com/appwrite/appwrite/pull/8520

### Fixes

* Fix migration stuck at "Starting Data Migration [...]" by @stnguyen90 in https://github.com/appwrite/appwrite/pull/8519

> Note: please run the [migrate command](https://appwrite.io/docs/upgrade#migration) even if upgrading from 1.5.x.

**Full Changelog**: https://github.com/appwrite/appwrite/compare/1.5.9...1.5.10

<hr /><em>This discussion was created from the release <a href='https://github.com/appwrite/appwrite/releases/tag/1.5.10'>1.5.10</a>.</em>

**Comment by CarloDotGG** [comment]
Hey Dream team!

Any idea when the https://appwrite.io/install/compose file can be updated to reflect the appwrite/appwrite:latest docker image so we can keep self-hosting  instances updated with latest? I keep getting a docker rate-limit issue with any version above 1.5.7 which seems like it might just need to be updated on your end.  Any assistance would be much appreciated :)
